.github/workflows/clojars.yml

```diff
@@ -0,0 +1,35 @@
+name: Clojure CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Prepare java
+        uses: actions/setup-java@v1
+        with: { java-version: 1.11 }
+
+      - name: Install clojure tools
+        uses: DeLaGuardo/setup-clojure@10.0
+        with:
+          cli: latest
+          bb: latest
+
+      - name: Build jar
+        run: bb uber
```

Comment:
Not seeing why you are using Babashaka to run clj/build commands, it seems unnecessary.